pages/inbox/browse.js

```diff
@@ -14,15 +14,39 @@ export const UserContext = createContext();
 
 export default function SingleReply() {
   const [showPopup, setShowPopup] = useState(false);
+  const [submitState, setSubmitState] = useState();
+  const [selectedSticker, setSelectedSticker] = useState();
+
   const router = useRouter();
   const { data: session } = useSession();
   const { replyid, letterid } = router.query;
   const { data: reply } = useSWR(`/api/replies/${replyid}`);
   const { data: letter } = useSWR(`/api/mails/${letterid}`);
 
+  async function handleOnSendSticker() {
+    const fetchReciever = await fetch(`/api/users/${reply.authorId}`, {
+      method: "GET",
+      headers: { "content-type": "application/json" },
+      //body: JSON.stringify({ emojis: selectedSticker }),
+    });
+    const Reciever = await fetchReciever.json();
+
+    const updateEmojiCollection = await fetch(`/api/users/${reply.authorId}`, {
+      method: "PATCH",
+      headers: { "content-type": "application/json" },
+      body: JSON.stringify({
+        stickers: [...Reciever.stickers, selectedSticker],
+      }),
+    });
```

Comment:
I think you can do this in one operation:

see: https://www.mongodb.com/docs/manual/reference/operator/update/push/

```suggestion
    const updatedUserResponse = await fetch(`/api/users/${reply.authorId}`, {
      method: "PATCH",
      headers: { "content-type": "application/json" },
      body: JSON.stringify({
        '$push': { 'stickers': selectedSticker },
      }),
    });
    
    // if you need the user afterwards:
    // const updatedUser = updatedUserResponse.json();
```